docs/tutorial.rst

```diff
@@ -144,7 +144,7 @@ and to calculate the analysis features:
     from ipfx.data_set_features import extract_data_set_features
 
     drop_failed_sweeps(data_set)     # sweeps with incomplete recording or failing QC criteria
-    cell_features, sweep_features, cell_record, sweep_records = extract_data_set_features(dataset)
+    cell_features, sweep_features, cell_record, sweep_records, cell_state = extract_data_set_features(dataset)
```

Comment:
It makes sense to separate the concerns and move record generation outside of this function. Then we would only have 3 output parameters.